.github/workflows/native-binary-build-dlight.nativeexecution.yml

```diff
@@ -0,0 +1,268 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# ----------------------
+#
+#  This workflow builds the native binaries for dlight.nativeexecution.
+#
+#  The following builds are made:
+#    - On a Linux X64 host :
+#          - Linux X64 binary
+#    - On a MacOS X64 host :
+#          - MacOS X64 binary
+#          - MacOS ARM64 binary
+#    - On a Windows Server X64 host (TODO) :
+#          - Windows X64 binary
+#
+#  ..meaning we build 32-bit binaries on 64-bit hosts.
+#
+#  The result of the build are files 'nativeexecution-external-sources.zip'
+#  and 'nativeexecution-external-binaries.zip' which can be downloaded from
+#  the GitHub Actions UI and prepared for release to be used by the Ant build
+#  scripts for the NetBeans distribution.
+#
+# ----------------------
+
+
+name: NetBeans Native Execution Libraries
+
+
+on:
+  # Triggers the workflow on push or pull request events but only for
+  # relevant paths
+  push:
+    paths:
+      - .github/workflows/native-binary-build-dlight.nativeexecution.y*
+      - ide/dlight.nativeexecution/**
+
+  pull_request:
+    paths:
+      - .github/workflows/native-binary-build-dlight.nativeexecution.y*
+      - ide/dlight.nativeexecution/**
+
+  # Allows you to run this workflow manually from the Actions tab in GitHub UI
+  workflow_dispatch:
+
+# cancel other PR workflow run in the same head-base group if it exists (e.g. during PR syncs)
+# if this is not a PR run (no github.head_ref and github.base_ref defined), use an UID as group
+concurrency: 
+  group: profiler-${{ github.head_ref || github.run_id }}-${{ github.base_ref }}
```

Comment:
you would have to change the `profiler-` prefix to something else, otherwise the two workflows might cancel each other if both run.

Reply:
Doh! 😄 I removed so many profiler references, I had to miss one. Will fix. Not that we need this so much as the job run is so short